resume_code/fetch_publications.py

```
@@ -0,0 +1,134 @@
+
+    output = {
+        "publications": pubs,
+        "metrics": {
+            "h_index": h_index,
+            "m_index": m_index,
+            "total_citations": total_citations,
+            "n_pubs": len(pubs),
+            "mean_cnci": mean_cnci,
+            "pct_top10": pct_top10,
+        },
+        "fetched_at": datetime.now().isoformat(),
```

Ruff (0.16.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)